src/test/java/org/fogbowcloud/manager/core/cloudconnector/LocalCloudConnectorTest.java

```diff
@@ -0,0 +1,359 @@
+		this.order = Mockito.mock(ComputeOrder.class);
+	    when(this.order.getType()).thenReturn(InstanceType.COMPUTE);
+	    //when(computePlugin.requestInstance(any(ComputeOrder.class), any(Token.class))).thenReturn(FAKE_INSTANCE_ID);
+		this.localCloudConnector.deleteInstance(order);
+	}
+	
+	@Test
+	public void testDeleteComputeInstance() throws FogbowManagerException, UnexpectedException {
+		this.order = Mockito.mock(ComputeOrder.class);
+	    when(this.order.getType()).thenReturn(InstanceType.COMPUTE);
+	    when(this.order.getInstanceId()).thenReturn(FAKE_INSTANCE_ID);
+		this.localCloudConnector.deleteInstance(order);
+	}
+	
+	@Test
+	public void testDeleteVolumeInstance() throws FogbowManagerException, UnexpectedException {
+		this.order = Mockito.mock(VolumeOrder.class);
+	    when(this.order.getType()).thenReturn(InstanceType.VOLUME);
+	    when(this.order.getInstanceId()).thenReturn(FAKE_INSTANCE_ID);
+		this.localCloudConnector.deleteInstance(order);
+	}
+	
+	@Test
+	public void testDeleteNetworkInstance() throws FogbowManagerException, UnexpectedException {
+		this.order = Mockito.mock(NetworkOrder.class);
+	    when(this.order.getType()).thenReturn(InstanceType.NETWORK);
+	    when(this.order.getInstanceId()).thenReturn(FAKE_INSTANCE_ID);
+		this.localCloudConnector.deleteInstance(order);
+	}
+	
+	@Test
+	public void testDeleteAttachmentInstance() throws FogbowManagerException, UnexpectedException {
+		this.order = Mockito.mock(AttachmentOrder.class);
+	    when(this.order.getType()).thenReturn(InstanceType.ATTACHMENT);
+	    when(this.order.getInstanceId()).thenReturn(FAKE_INSTANCE_ID);
+		this.localCloudConnector.deleteInstance(order);
+	}
+	
+	@Test
+	public void testgetImage() throws FogbowManagerException, UnexpectedException {
+	    when(this.imagePlugin.getImage(any(String.class), any(Token.class))).thenReturn(this.image);
+		assertEquals(FAKE_IMAGE_ID, this.localCloudConnector.getImage(FAKE_IMAGE_ID, federationUser).getId());
+	}
+	
+	@Test
+	public void testgetNullImage() throws FogbowManagerException, UnexpectedException {
+	    when(this.imagePlugin.getImage(any(String.class), any(Token.class))).thenReturn(null);
+		assertNull(this.localCloudConnector.getImage(FAKE_IMAGE_ID, federationUser));
+	}
+	
+	@Test
+	public void testGetUserComputeQuota() throws FogbowManagerException, UnexpectedException {
+		
+		ComputeAllocation fakeTotalComputeAllocation = new ComputeAllocation(2, 2048, 2);
+		ComputeAllocation fakeUsedComputeAllocation = new ComputeAllocation(1, 1024, 1);
+		ComputeQuota fakeComputeQuota = new ComputeQuota(fakeTotalComputeAllocation, fakeUsedComputeAllocation);
+	    when(this.computeQuotaPlugin.getUserQuota(any(Token.class))).thenReturn(fakeComputeQuota);
+	    
+	    ComputeQuota quota = (ComputeQuota) this.localCloudConnector.getUserQuota(federationUser, InstanceType.COMPUTE);
+	    assertEquals(2, quota.getTotalQuota().getvCPU());
```

Comment:
instead of using literal, e.g 2, 2018 and 2 in line 318 and 325, replace them by a variable with a good name

Reply:
Done